docs/developer-center/new-guides/quickstart.md

```diff
@@ -0,0 +1,75 @@
+## Quickstart
+
+Hi! Glad to see you around! This guide walks you through the common steps a data scientist has to follow in their spatial analyses and how CARTOframes can help in each of them. We will follow a real use case to show it. We encourage you to have your environment ready so you can learn how to use the library by using it. If you haven't done it yet, check this guide.
+
+### Real use case
+
+Imagine that you are a data scientist at a bikesharing company and you want to know how well the company has placed their stations. Apart from that, you would also like to recommend new spots so the company can continue growing. Before starting, you list the steps you have to follow to achieve it:
```

Comment:
Let's say you work for a bike-sharing company that asks you for recommendations on where to open new spots. To begin with, you wonder how well-positioned their stations are. So before starting, you list the steps you have to follow to achieve it: